content/updates/opentofu-early-adopters.md

```diff
@@ -0,0 +1,104 @@
+2. We had to update the `terraform-spacelift-cloud-infrastructure-automation` module to support setting the ["workflow tool" argument](https://registry.terraform.io/providers/spacelift-io/spacelift/latest/docs/resources/stack#terraform_workflow_tool) to a new `OPEN_TOFU` value. After we upgraded our internal usage of that module, this change enabled us to designate OpenTofu as our preferred tool, instead of Terraform. [The open source PR for that module change was made here](https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation/pull/162).
+3. We had to update our atmos configuration to ensure that atmos commands executed `tofu` and not `terraform`.
+4. Critically, we had to pass the `OPEN_TOFU` workflow tool value to Spacelift. This required the following changes in our base atmos `_defaults.yaml` file:
+
+```yaml
+terraform:
+  command: tofu
+  settings:
+    spacelift:
+
+      # Tofu > Terraform
+      terraform_workflow_tool: OPEN_TOFU
+
+```
+
+![Pull Request Description for my changes to introduce OpenTofu](/img/updates/opentofu-early-adopters/pr-header.png "Pull Request Description for my changes to introduce OpenTofu")
+
+And that was it! Once that PR got merged, all of our Spacelift Automation and local usage of Atmos changed over to  OpenTofu and we were done! 🏁
+
+### How Long Did It Take?
+
+Sure, the changes were minimal, but how long did it really take to implement them?
+
+As mentioned above, we manage thousands of resources, over 90 workspaces, 39,000+ lines of code over nearly 500 files. But due to the automation Masterpoint has built and compatibility of OpenTofu, this migration took just over **7 hours** of work across multiple days. I personally did this migration work myself, and I was pleasantly surprised and happy with how quickly it all came together.
+
+### How Has It Been Going?
+
+Our migration journey has been very positive. We’ve only hit a couple minor issues that were quickly addressed:
+
+1. GPG Key signing support for non-mainstream providers is still a WIP. This results in tiny warnings like the screenshot; we find these to be an eyesore and prefer our logs to be clean to avoid missing any true issues. Luckily, for such providers, we’ve opened issues and it appears these will be fixed very soon:
+    1. [https://github.com/carlpett/terraform-provider-sops/issues/115](https://github.com/carlpett/terraform-provider-sops/issues/115) – Fixed!
+    2. [https://github.com/cloudposse/terraform-provider-utils/issues/344](https://github.com/cloudposse/terraform-provider-utils/issues/344) ([@kevcube](https://github.com/kevcube) is a Masterpoint engineer) – Fixed!
+    3. [https://github.com/tailscale/terraform-provider-tailscale/issues/335](https://github.com/tailscale/terraform-provider-tailscale/issues/335) – Open (as of the time of writing)
+
+![An example GPG Key Warning](/img/updates/opentofu-early-adopters/gpg-key-warning.png "An example GPG Key Warning")
+
```

Comment:
The list ordering is broken. I believe we need to remove these empty lines and add indentation before the image.

Reply:
Great catch!